views/dashboard.hbs

```diff
@@ -0,0 +1,107 @@
+                <div class="col-md-1 toolbar-elements">Profile</div>
+                <!-- dropdown button dummy -->
+                <!-- add and arrow svg or png to show the dropdown properly-->
+                <div class="col-md-1 toolbar-elements dropdown show">
+                    <a class="btn btn-outline-light dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> ></a>
+                    <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+                        <a class="dropdown-item" href="#">Action</a>
+                        <a class="dropdown-item" href="#">Another action</a>
+                        <a class="dropdown-item" href="#">Something else here</a>
+                    </div>
+                </div>
+            </div>
+            <br> 
+            <!-- TaskManager-->
+            <div id="taskManager">
+                <!-- Row 2 -->
+            <div class="row">
+                <!-- Task Manager container TODO give ids to these buttons -->
+                <div class="col-md-2 taskManagerContainer">
+                    <button type="button" class="buttonPadding btn btn-outline-light btn-lg btn-block">   Blog Post  </button>
+                    <button type="button" class="buttonPadding btn btn-outline-light btn-lg btn-block">   Add Projects</button>
+                    <button type="button" class="buttonPadding btn btn-outline-light btn-lg btn-block  dropdown-toggle " data-toggle="dropdown">Add Templates</button>
+                    <button type="button" class="buttonPadding btn btn-outline-light btn-lg btn-block  dropdown-toggle " data-toggle="dropdown">Statistics</button>
+                </div>
+                <!-- Content break -->
+                <div class="contentBreak"></div>
+                <!-- Card container begins here-->
+                <!-- Todo Add ids here so you can load more cards via Js and then add it as a template-->
+                <div class="col-md-7">
+                    <div class="CardContainer " style="background-color: rgb(253, 253, 253);">
+                        <div class="CardContentImage"> <img src="http://via.placeholder.com/200x200" alt="404Notfound" align="right"> </div>
+                        <div class="CardTitle">Project 1</div>
+                        <div class="CardContentText">
+                            <p>
+                                Lorem ipsum dolor sit amet, ius ornatus adolescens cu, te homero laudem cum. Sea accusata evertitur cu. Pro et admodum tractatos, mea id nisl error deserunt, mel meliore accusam id. Feugiat commune nam ei, ei quem quot munere duo. Eam ut mucius euismod volumus, eum in saepe pertinax, nam eu harum saepe oporteat.
+                            </p>
+                        </div> 
+                    </div> 
+                    <div class="CardContainer " style="background-color: rgb(253, 253, 253);">
```

Comment:
avoid inline styles

Reply:
again I did try to avoid it but for some reason, bootstrap was overriding my CSS class colour
but taking its an initial mockup.

so sorry about that I had to try.

if possible could you see what could be going wrong i would appriciate it.

Reply:
yes I can look into it.

Reply:
@SMahil 
I would absolutely appreciate this cause i am used to C++ OOP and have no idea what is going on here.